db/seeds.rb

```diff
@@ -5,3 +5,7 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+
+Venue.create( name: 'House of Blues', address: '15 Lansdown Street', description: 'house o blues yo', capacity: 200, url: 'http://www.houseofblues.com/' )
```

Comment:
given this is in a seed script, we should make use of the `create!` method so that the script will fail if creation of the Venue does not pass validation